docs/tutorial/correlatedstacks.rst

```diff
@@ -24,6 +24,14 @@ from Bluelake if available. This functionality can be turned off with the option
 
     stack2 = lk.CorrelatedStack("example2.tiff", align=False)
 
+You can obtain the image stack data as a `numpy` array using the `get_image()` method::
+
+    red_data = stack.get_image(channel="red") # shape = [y_pixels, x_pixels, n_frames]
+    rgb_data = stack.get_image(channel="rgb") # shape = [y_pixels, x_pixels, 3 channels, n_frames]
```

Comment:
Is there a reason to deviate from `Scan` here? Multiframe scans are stored as `n_frames`, `pixel`, `pixel`, `colors`.

Reply:
the reason is that I meant to make sure it was in line with `Scan` but somehow permuted through every other combination of axes before finally getting it right.....